src/MSBuild/XMake.cs

```diff
@@ -2563,7 +2573,8 @@ private static bool GatherAutoResponseFileSwitchesFromFullPath(string autoRespon
                     getProperty = commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.GetProperty] ?? Array.Empty<string>();
                     getItem = commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.GetItem] ?? Array.Empty<string>();
                     getTargetResult = commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.GetTargetResult] ?? Array.Empty<string>();
-                    if (getProperty.Length > 0 || getItem.Length > 0 || getTargetResult.Length > 0)
+                    getResultOutputFile = commandLineSwitches[CommandLineSwitches.ParameterizedSwitch.GetResultOutputFile].FirstOrDefault() ?? string.Empty;
+                    if ((getProperty.Length > 0 || getItem.Length > 0 || getTargetResult.Length > 0) && getResultOutputFile.Length == 0)
```

Comment:
suggestion: assign this to a variable, something like `printEvaluationToStdout`.

Reply:
It doesn't just control evaluation but also build information. I'm proposing minimizeStdOutOutput. What do you think?